### DISCUSSION TITLE
Is downgrading school chromebook from 108 to 105 worth it?

### DISCUSSION THREAD

**Discussion by 1-Mythical** [3kh0/ext-remover#473] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I need the find a method and I heard ltbeef and ingot work in versions before 106. I am planning to also update it back to 108 near the end of the year. Will everything be the same. (enrollment, account, extensions, etc.)

**Comment by 1vers1on** [comment, marked as answer]
no. just use killcurly

**Reply by 1-Mythical** [reply]
Doesn't work for me. I was asking if it is worth it to downgrade. Will enrollment still work and it be a normal school chromebook on 105 rather than 108

**Reply by 1-Mythical** [reply]
Did it anyways and can confirm. It is worth it.

**Reply by Aka-but-nice** [reply]
Yes, I can confirm. Reasons being you'll have access to ingot, and possibly other exploits, which got blocked, I recommend, if you do downgrade, use <a href="https://caub.glitch.me">caub</a> to stay downgraded, and you'll still be enrolled.

**Reply by 1-Mythical** [reply]
Thank you. I have downgraded to 102 and my friends will get 105 soon. Can the school find out about this and get my friends and I in trouble? Also, caub wasn't needed because my Chromebook doesn't auto-update. I test by manually get 109 and then downgraded back to 102.

**Reply by MicroChipKali** [reply]
105 is the best option because in my school district lets me use an extension blocker and terminal.

**Reply by ehunt34** [reply]
101 or lower would be best as you have access to the set_cellular_ppp exploit.

**Comment by rwg100** [comment]
downgrading is illegal, dont downgrade, its not worth it.

**Reply by 1-Mythical** [reply]
Why is an option here on the github page then? Also, what happens when the school finds out you downgraded? I don't remember hearing about it not being legal.
